packages/gatsby/src/db/nodes.js

```diff
@@ -2,7 +2,7 @@
 const _ = require(`lodash`)
 const { store } = require(`../redux`)
 const nodesDb: NodeStore = require(`../redux/nodes`)
-const runQuery = require(`../redux/run-sift`).runSift
+const { runFastFiltersAndSort } = require(`../redux/run-sift`)
```

Comment:
should we rename the file from `run-sift` to something else?

Reply:
Yes, working on that. Going to do this as part of migrating it to TS :)